profiles/seccomp/default_linux.go

```diff
@@ -605,6 +602,9 @@ func DefaultProfile() *Seccomp {
 					"fsopen",
 					"fspick",
 					"lookup_dcookie",
+					"lsm_get_self_attr", // kernel v6.8, libseccomp v2.6.0
+					"lsm_list_modules",  // kernel v6.8, libseccomp v2.6.0
```

Comment:
Without having given it much thought, and based on names, `get`, and `list` seem potentially safe (but not sure how software handles half of these being blocked, and the other half not blocked).


@AkihiroSuda @tonistiigi for thoughts here.